src/sqlparser/src/ast/mod.rs

```diff
@@ -557,24 +551,6 @@ impl fmt::Display for Expr {
     }
 }
 
-/// Wrap complex expressions with necessary parentheses.
-/// For example, `a > b LIKE c` becomes `a > (b LIKE c)`.
-fn fmt_expr_with_paren(e: &Expr) -> String {
```

Comment:
I'm also curious why the upstream adds this extra step. 🤔 Is it just for better readability after formatting?